dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -779,6 +785,54 @@ ERROR_SINGLE_REPO_CONNECTION_FAILED=CWWKF1389E: Unable to create a JSON file at
+ERROR_FAILED_TO_DOWNLOAD_ASSETS_FROM_MAVEN_REPO.useraction=Ensure that the repository contains the asset, and run the command again.
+
+ERROR_COULD_NOT_DETERMINE_RUNTIME_PROPERTIES_FILE=CWWKF1394E: Could not determine the Liberty runtime version.
+ERROR_COULD_NOT_DETERMINE_RUNTIME_PROPERTIES_FILE.explanation=The properties file {0} may be missing or corrupted.
+ERROR_COULD_NOT_DETERMINE_RUNTIME_PROPERTIES_FILE.useraction=Ensure the properties file exists.
+
+ERROR_MAVEN_COORDINATE_WRONG_VERSION=CWWKF1395E: Cannot install feature with Maven coordinate {0} onto a {1} Liberty runtime version.
+ERROR_MAVEN_COORDINATE_WRONG_VERSION.explanation=The feature is incompatible with the Liberty runtime.
+ERROR_MAVEN_COORDINATE_WRONG_VERSION.useraction=Specify a valid version for the feature.
+
+ERROR_MAVEN_COORDINATE_WRONG_PACKAGING=CWWKF1396E: The packaging format in feature Maven coordinate {0} is invalid. The valid packaging format is esa.
+ERROR_MAVEN_COORDINATE_WRONG_PACKAGING.explanation=The feature is incompatible with the Liberty runtime.
+ERROR_MAVEN_COORDINATE_WRONG_PACKAGING.useraction=Specify "esa" for the packaging in the Maven coordinate.
+
+ERROR_MAVEN_COORDINATE_INVALID=CWWKF1397E: Invalid maven coordinate provided: {0}
+ERROR_MAVEN_COORDINATE_INVALID.explanation=The maven coordinated provided is not in the correct format.
+ERROR_MAVEN_COORDINATE_INVALID.useraction=Enter a valid maven coordinate.
+
+ERROR_SERVER_NOT_EXIST=CWWKF1398E: The server {0} does not exist. Please enter a valid server name and try again.
+ERROR_SERVER_NOT_EXIST.explanation=The server does not exist.
+ERROR_SERVER_NOT_EXIST.useraction=Create the server and run the command again.
+
+ERROR_MAVEN_JSON_NOT_FOUND=CWWKF1399E: The group id {0} is invalid. Ensure that the valid group id for the feature coordinate is provided or the system can access the internet and the configured repository.
+ERROR_MAVEN_JSON_NOT_FOUND.explanation=The features json file could not be found on the Maven repository.
+ERROR_MAVEN_JSON_NOT_FOUND.useraction=Check your network connection and ensure the features json is hosted on the Maven repository.
+
+ERROR_IMPROPER_HTTPPROXY_FORMAT=CWWKF1400E: Invalid http_proxy environment variable format {0}. Ensure that the http_proxy variable is in the following format: http_proxy=http://<proxy-host>:<proxy-port>
+ERROR_IMPROPER_HTTPPROXY_FORMAT.explanation=Invalid proxy environment variable format.
+ERROR_IMPROPER_HTTPPROXY_FORMAT.useraction=Reformat the http_proxy environment variable to fit specifications, namely http_proxy=http://<proxy-host>:<proxy-port>.
+
+ERROR_IMPROPER_HTTPSPROXY_FORMAT=CWWKF1401E: Invalid https_proxy environment variable format {0}. Ensure that the https_proxy variable is in the following format: https_proxy=https://<user>:<password>@<proxy-host>:<proxy-port>
```

Comment:
Change
"Invalid http_proxy environment variable format {0}. Ensure that the http_proxy variable is in the following format: http_proxy=http://:"
To
"The format of the {0} http_proxy environment variable is invalid. Ensure that the http_proxy variable is in the http_proxy=http://: format. "